projects/packages/ip/package.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
It seems unlikely we'll need this file. If we were to want a package with IP functions for JS, we'd likely make a separate one in js-packages.

Reply:
Addressed via https://github.com/Automattic/jetpack/pull/28765/commits/643db8ccaee40ba2472c62fcb32c76341f9351b8 :+1: :+1: